talk/concurrency/threadlocal.tex

```diff
@@ -0,0 +1,22 @@
+\subsection[TLS]{Thread-local storage}
+
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Thread-local storage}
+  \begin{block}{\mintinline{cpp}{thread_local} keyword}
+    \begin{itemize}
+      \item When declaring a variable thread-local, every thread will have its own copy
+      \item Most useful for things treated as ``working memory'' in each thread
+      \item Note: Can also be declared directly on the stack of a thread (faster)
```

Comment:
```suggestion
      \item Note: Can also be declared directly on the stack of a thread
      \begin{itemize}
        \item and will be faster, thus should be preferred when possible
      \end{itemize}
```

Reply:
you might also want to mention that `thread_local` variables are (usually) implicitly `static` -- if you think about it it must be that way, but if you don't think about it you'll get surprising behavior :)

Reply:
`thread_local` implies `static` for variables at block scope (which in my experience is the situation in which you most commonly use `thread_local`)